dev/com.ibm.websphere.appserver.features/visibility/public/socialLogin-1.0/resources/l10n/com.ibm.websphere.appserver.socialLogin-1.0.properties

```diff
@@ -16,8 +16,4 @@
 
 description=Social login is a form of single sign-on (SSO) that enables users to sign in to a \
 secured website by using their existing social media account instead of creating an account \
-specifically for the website. For example, you can configure social login so that users \
-can log in to your website with their Facebook or Twitter accounts. \
-You can enable social login for any social media platform that uses \
-the OAuth 2.0 or OpenID Connect 1.0 standard for authorization. \
-It can be used instead of, or in addition to, the configured user registry.
+specifically for the website.
```

Comment:
although this edit works, the existing sentence it completes is too long (35 words) for style guidance. Is it possible to remove the final clause?

`Social login is a form of single sign-on (SSO) that enables users to sign in to a secured website by using their existing social media account.`

Alternatively, could break into two sentences (though I think that was what this edit was trying to avoid)

`Social login is a form of single sign-on (SSO). This feature enables users to sign in to a 
secured website by using their existing social media account instead of creating an account specifically for the website.`

Reply:
I can look at rewording it but I make it about 25-26 words, not 35...

Reply:
that should satisfy the acrolinx requirements

Reply:
this would work length-wise-
`Social login enables users to sign in to a website by using their existing social media account instead of creating an account specifically for the website.`

but would lose the mention of SSO

Reply:
I think the "SSO" bit is important for someone who already knows what SSO is (as a recognisable term). What's the max?

`Social login is a form of single sign-on (SSO) that enables users to sign in to a secured website with their existing social media account.` would work though

Reply:
I believe it's 30 words

Reply:
Okay, I've pushed a shorter version.